src/conf/domain_conf.c

```diff
@@ -19663,6 +19663,10 @@ virDomainDefParseXML(xmlXPathContextPtr ctxt,
     g_autofree xmlNodePtr *nodes = NULL;
     g_autofree char *tmp = NULL;
     g_autoptr(virDomainDef) def = NULL;
+    int max_available_isa_serial_ports = 4;
+    long long int used_serial_port_buffer = 0;
```

Comment:
this can be smaller like uint_8

Reply:
Yeah noted, wanted to define it as a enum/macro/gflag. Not sure about the proper location where to define it. Let me define it in the same file. What do you say ?